lib/resources/r4/charge_item.yaml

```diff
@@ -105,3 +105,17 @@ fields:
         - display: Bill Code Schedule
           system: https://fhir.cerner.com/&lt;EHR source id&gt;/codeSet/14002
           info_link: https://fhir.cerner.com/millennium/r4/proprietary-codes-and-systems/#code-set-14002-bill-code-schedule
+
+  - name: performer.function
+    required: 'No'
+    type: performerFunction
```

Comment:
If we are specifying performer.function we should set the type to CodeableConcept.

Reply:
Yes this is not applicable now after changes made for this below comment.
https://github.com/cerner/fhir.cerner.com/pull/682#discussion_r663179455